### ISSUE TITLE
Import methods from lodash for tree-shaking 

### ISSUE THREAD

**Issue by nite** [rekit/antd-form-builder#5]
I've noticed FormBuilder.js has `import _ from 'lodash'` - this will import all of lodash. 
You might want to do things like this to enable tree-shaking. 

```
import memoize from 'lodash/memoize;
import isArray from 'lodash/isArray;
import pick from 'lodash/pick;
import capitalize from 'lodash/capitalize;
import has from 'lodash/has;
import get from 'lodash/get;
import find from 'lodash/find;
```
then just replace `_.` with `''`

https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

**Comment by supnate** [comment]
Thanks @nite for the suggestion. However `lodash` is also heavily used in `antd`, so this is not a bottle neck.

**Comment by nite** [comment]
Do they not tree-shake themselves? Havent dug into the code yet.

**Comment by supnate** [comment]
fixed in 2.0.
